lib/routes/subswap/swap/widgets/inprogress_swap.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:breez_sdk/bridge_generated.dart';
+import 'package:breez_translations/breez_translations_locales.dart';
+import 'package:c_breez/services/injector.dart';
+import 'package:c_breez/widgets/flushbar.dart';
+import 'package:c_breez/widgets/link_launcher.dart';
+import 'package:flutter/material.dart';
+
+class SwapInprogress extends StatelessWidget {
+  final SwapInfo swap;
+
+  const SwapInprogress({
+    required this.swap,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final texts = context.texts();    
+    final List<String> swapTransactions = [];
+    swapTransactions.addAll(swap.unconfirmedTxIds);
+    swapTransactions.addAll(swap.unconfirmedTxIds);
```

Comment:
Why the double call here? Is it by mistake or a workaround for a bug?
Also, since swap isn't being used elsewhere, I suggest we populate the swapTransactions where we define SwapInprogress widget.
`SwapInprogress(swapTransactions: swapState.inProgress!.unconfirmedTxIds.toList())`

!!

I've checked the respective breez-sdk PR and see that unconfirmedTxIds is already a List<String>. We can use it directly.
`SwapInprogress(swapTransactions: swapState.inProgress!.unconfirmedTxIds)`

Reply:
I did a mistake indeed. Should be a concatenation  of both `unconfirmedTxIds` and `confirmedTxIds` since we consider it in progress as long as has utxos (confirmed or unconfirmed).
I will fix.

Reply:
@erdemyerebasmaz  I pushed changes which also adapt the page to the changes in https://github.com/breez/breez-sdk/pull/84

Reply:
> I did a mistake indeed. Should be a concatenation of both `unconfirmedTxIds` and `confirmedTxIds` since we consider it in progress as long as has utxos (confirmed or unconfirmed).
> I will fix.

I'm only seeing unconfirmedTxIds being used, it's not using a concatenation of both.

Reply:
Right. I did that because this was the behavior in breez mobile but actually concatenating them adds to the user experience as the user can see the confirmed transactions in case of redeem error, so I pushed that.
Thanks.